sensor_msgs/CompressedImage image
---
corobot_openpose/OpenPoseHumanList human_list